src/api/teams/controllers/add-user-to-team.js

```diff
@@ -23,19 +22,19 @@ const addUserToTeamController = {
     const dbTeam = await getTeam(request.db, teamId)
     const dbUser = await getUser(request.db, userId)
 
-    if (isNull(dbTeam) || isNull(dbUser)) {
-      throw Boom.notFound('User or Team not found')
+    if (!dbTeam) {
+      throw Boom.notFound('Team not found')
+    } else if (!dbUser) {
+      throw Boom.notFound('User not found')
     } else if (teamHasUser(dbTeam, dbUser)) {
       throw Boom.conflict('User already a member of the team')
```

Comment:
I know this wasn't changed in this PR. But why boom conflict  if already there. idempotency and all that. :shrug:

Reply:
Agreed - have updated